modules/web/middleware/locale.go

```diff
@@ -42,8 +43,30 @@ func Locale(resp http.ResponseWriter, req *http.Request) translation.Locale {
 	}
 
 	if changeLang {
-		SetCookie(resp, "lang", lang, 1<<31-1)
+		SetLocaleCookie(resp, lang)
```

Comment:
This is a potential change we need to be sure I'm correct here.

Reply:
What was this `1<<31-1` for?

Reply:
So previously clicking on the language selector at the bottom of the screen would set a permanent lang cookie for unlogged in users ensuring that they always got their correct language. 

It's kinda a bad thing to set a permanent cookie and there moans about it in one of the issues.

TBH if you're not logged in you shouldn't necessarily get to have permanent settings but I'm not sure.

Reply:
If you have selected language why would you not want to keep it? This way login will be pretty much always in not the language you selected

Reply:
I'll put it back in and we should look at this another time.